packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -1050,6 +1050,7 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
                   disabled={toolbarDisabled}
                   icon={vsKebabVertical}
                   tooltip="More Actions..."
+                  onClick={() => undefined}
```

Comment:
```suggestion
                  onClick={() => {
                    // no-op: click is handled in `DropdownMenu`
                  }}
```
To keep it consistent with other Buttons that use DropdownMenus